modules/administration/pages/auth-methods-sso-example.adoc

```diff
@@ -40,6 +40,76 @@ java.sso.onelogin.saml2.idp.single_logout_service.url
 java.sso.onelogin.saml2.idp.x509cert
 ----
 
+Here is an example of the final SUSE Manage's rhn.conf:
+
++
+----
+java.sso = true
+
+
+# This is the configuration file for Single Sign-On (SSO) via SAMLv2 protocol
+# To enable SSO, set java.sso = true in /etc/rhn/rhn.conf
+#
+# Mandatory changes: search this file for:
+# - YOUR-PRODUCT
+# - YOUR-IDP-ENTITY
+#
+# See product documentation and the comments inline in this file for more
+# information about every parameter.
+#
+#
+#
+#
+# If 'strict' is True, then the Java Toolkit will reject unsigned
+# or unencrypted messages if it expects them signed or encrypted
+# Also will reject the messages if not strictly follow the SAML
+#
+# WARNING: In production, this parameter setting parameter MUST be set as "true".
+# Otherwise your environment is not secure and will be exposed to attacks.
+# Enable debug mode (to print errors)
+# Identifier of the SP entity (must be a URI)
+java.sso.onelogin.saml2.sp.entityid = https://suma-long-42-srv.mgr.suse.de/rhn/manager/sso/metadata
```

Comment:
Should we be using our reference servers in our examples? 

Per the SUSE style guide: Use http://www.example.com/ and http://www.example.org/ as example domains. Both domains were registered for use in documentation.

Reply:
It is admittedly a bit awkward with the identity server. We should probably come up with our own terms for domains and servers names. From a user perspective though, the usecase may not be very clear based on our infra.